src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -338,33 +338,37 @@
   "item.minecolonies.golden_bread": "Golden Bread",
   "item.minecolonies.chorus_bread": "Chorus Fruit Bread",
 
-  "item.minecolonies.scroll_tp": "Ultrasafe colony teleport scroll",
-  "item.minecolonies.scroll_tp.tip": "Safely teleports you to the set colony",
+  "item.minecolonies.scroll_tp": "Colony Portal Scroll",
+  "item.minecolonies.scroll_tp.tip": "Teleports you to the set colony",
   "item.minecolonies.scroll.colony.tip": "Colony: %s",
-  "item.minecolonies.scroll.colony.none": "None, set one with sneak + right-click",
-  "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
-  "item.minecolonies.scroll_buff": "Sacred Scroll of Regeneration",
+  "item.minecolonies.scroll.colony.none": "None, set one with sneak + use",
+  "item.minecolonies.scroll_area_tp": "Colony Group Portal Scroll",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports you and nearby players to the colony",
+  "item.minecolonies.scroll_buff": "Regeneration Scroll",
   "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",
-  "item.minecolonies.scroll_guard_help": "Spatial guard reinforcement scroll",
-  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time, wololo!",
-  "item.minecolonies.scroll_highlight": "Worker-where-are-you scroll",
-  "item.minecolonies.scroll_highlight.tip": "Sneak-Use on a minecolonies building, to highlight and speed up its workers",
+  "item.minecolonies.scroll_guard_help": "Reinforcements Scroll",
+  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time. Wololo!",
+  "item.minecolonies.scroll_highlight": "Worker Locator Scroll",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + use on a hut block to highlight and speed up its worker(s)",
   "minecolonies.scroll.nopermission": "You do not have the required permissions for this colony.",
-  "minecolonies.scroll.needcolony": "Please register this to a colony first with shift + right-clicking a building.",
-  "minecolonies.scroll.noguardbuilding": "Please register this to a guard building by shift + right-clicking it.",
-  "minecolonies.scroll.wrongdim": "Can only be used in the same dimension as the colony.",
-  "minecolonies.scroll.registered": "Registered scroll with colony %s",
+  "minecolonies.scroll.needcolony": "Please register this to a colony first by sneak + using a hut block.",
```

We're using the scroll, not the hut block :)
```suggestion
  "minecolonies.scroll.needcolony": "Please register this to a colony first by sneak + using it on a hut block.",
```